RocketChat/Rocket.Chat.ReactNative
# [CHORE] Use JSON files for i18n

## Proposed changes

Use JSON files instead of JS objects for i18n.

## Issue(s)	


## How to test or reproduce

- Set a new language and the app should apply it as usual
- Set a RTL language and the app should work as well

## Screenshots

## Types of changes




## Checklist



## Further comments
